lib/y2lan_restart_common.pm

```diff
@@ -47,7 +47,11 @@ sub initialize_y2lan {
 sub open_network_settings {
     type_string "yast2 lan\n";
     accept_warning_network_manager_default;
-    assert_screen 'yast2_lan', 100;       # yast2 lan overview tab
+    assert_screen([qw(yast2_lan yast2_lan-bsc1004643)], 100);    # yast2 lan overview tab
```

Comment:
As we have discussed in person, let's simply create workaround needle with bug reference and press home unconditionally, so test module works with multiple interfaces. Other thing we need `send_key_until_needle_match` to select correct one. WDYT?

Reply:
Yes, right! we avoid to remove the soft-failure as in this case to send key home is not specific from the workaround and the needle is enough. I changed the needles to yast2_lan to match as we created initially.